Sources/Core/AST/Stmt/ConditionalCompilationStmt.swift

```diff
@@ -94,10 +94,63 @@ public struct ConditionalCompilationStmt: Stmt {
 
   }
 
+  /// Abstracts the whole condition structure and its sub-conditions.
```

Comment:
I think there is a way to improve this docs.
```suggestion
  /// A condition in a conditional compilation statement expressed as a composition of predicates.
```
But at the same time it occurred to me that we could probably modify `Condition` rather than introducing a new abstraction. That's because the most natural way to document this thing is to say that a condition is a propositional statement (as in propositional logic). The propositions are the factors.